src/components/Homepage.jsx

```diff
@@ -37,6 +65,33 @@ const Homepage = () => {
 
   return (
     <section>
+      <div className="sorting">
+        <select value={sortingOptions.sort_by} onChange={handleChange}>
+          <option value="created_at"> Sort by: Date</option>
+          <option value="comment_count"> Sort by: Comment Count</option>
+          <option value="votes"> Sort by: Votes</option>
+        </select>
+        <div>
+          <img
+            src={
+              sortingOptions.order === "desc" ? IconArrowUpActive : IconArrowUp
```

Comment:
Nice conditional logic with the styling here! For accessibility reason I'd make this a button not a div as it's performing an action (you can remove default stylings, have a quick google to look at what properties need resetting)